tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "experimentalDecorators": true,
     "noImplicitAny": true,
     "listFiles": false,
-    "noLib": false
+    "noLib": false,
+    "outDir": "./ng2-bootstrap"
```

Comment:
nope, this will not work as expected,
consumers of this lib will have to
`import 'ng2-bootstrap/ng2-bootstrap/ng2-bootstrap' to get to .js files